### PR TITLE
Bug fix party

### PR DESCRIPTION
**Pull request by rohitdhamankar**
some bug fixes + manual resizing

### CODE REVIEW

**Comment by shivampdesai** [comment]
You might wanna update the ui files because I got a seg fault when I opened a file since it's probably not compatible with the new UI

**Comment by shivampdesai** [comment]
Also a suggestion: it looks a little cluttered with all objects having the resize icons, it might help to only have them for an object that's selected

File: sym_object.py [hunk 1]
```diff
@@ -583,22 +577,22 @@ def deleteButtonPressed(self, event):
...
-        r2_x = item.scenePos().x() + item.width
-        r2_y = item.scenePos().y() + item.height
+        l1_x = self.mapToScene(self.rect.rect()).boundingRect().left()
+        l1_y = self.mapToScene(self.rect.rect()).boundingRect().top()
+        r1_x = self.mapToScene(self.rect.rect()).boundingRect().right()
+        r1_y = self.mapToScene(self.rect.rect()).boundingRect().bottom()
+        l2_x = item.mapToScene(item.rect.rect()).boundingRect().left()
+        l2_y = item.mapToScene(item.rect.rect()).boundingRect().top()
+        r2_x = item.mapToScene(item.rect.rect()).boundingRect().right()
+        r2_y = item.mapToScene(item.rect.rect()).boundingRect().bottom()
```
Comment (afarooqui98) [inline sym_object.py:595]: put this in its own function? just a suggestion

Comment (rohitdhamankar) [reply sym_object.py]: good thinking

**Pull request merged**